.cargo/config.toml

```diff
@@ -9,4 +9,7 @@ rustflags = [
 rustflags = [
     "-C", "link-arg=-undefined",
     "-C", "link-arg=dynamic_lookup",
-]
\ No newline at end of file
+]
+
+[alias]
+xtask = "run -p xtask --"
```

Comment:
see https://github.com/matklad/cargo-xtask 

this seems like it's on defacto standard in the rust ecosystem for repo automation stuff